File: js_algo/insersionSort.js
function insersionSort(arr){
    startSort = 1
    while(startSort < arr.length){
        let currentValue = arr[startSort]
        let index = startSort
        while(index > 0 && currentValue < arr[index - 1]){
            arr[index] = arr[index - 1]
            index-=1;
        }
        arr[index] = currentValue
        startSort += 1
    }
}

const arr = [43, 24, 45, 27, 35, 47, 22, 48]

insersionSort(arr)

console.log(arr)

File: insersion_sort.py
def insertion_sort(arr):
    end = len(arr)
    endSorted = 1
    while endSorted < end:
        currentValue = arr[endSorted]
        index = endSorted
        while index > 0 and currentValue < arr[index - 1]:
            arr[index] = arr[index - 1]
            index = index - 1
        arr[index] = currentValue
        endSorted = endSorted + 1


arr = [43, 24, 45, 27, 35, 47, 22, 48]

insertion_sort(arr)

#print(arr)